smk/dependencies.smk

```diff
@@ -14,7 +14,7 @@ include: 'include/cmdline_validator.smk'
 
 script_dir=workflow.basedir+"/../scripts"
 
-metaphlan_index = 'mpa_vOct22_CHOCOPhlAnSGB_202212'
+metaphlan_index = 'mpa_vJun23_CHOCOPhlAnSGB_202403'
 metaphlan_version = '4.0.6'
```

Comment:
From what I can see, metaphlan relies on mpa_latest file to tell, which database it should use, if none are specified with `--index`.  For clarity's sake, we should either organise the databases into separate subfolders beneath the metaphlan version, or specify `--index` in QC_2.

Reply:
And could you please change the line in versions.smk
`echo "metaphlan database $(basename $(ls {minto_dir}/data/metaphlan/{metaphlan_version}/*.csv) | sed 's|_VINFO.csv||')" >> $VOUT`
to
`echo "metaphlan database $(cat {minto_dir}/data/metaphlan/{metaphlan_version}/mpa_latest)" >> $VOUT`
to account for multiple index versions.

Reply:
Rule `metaphlan_tax_profile` in `QC_2.smk` has had `--index {metaphlan_index}` for sometime already. So it explicitly asks for the given index.

I fixed the line in `versions.smk`.